Makefile

```diff
@@ -67,6 +67,11 @@ test:
 test-all: export RECONCILER_INTEGRATION_TESTS = 1
 test-all: test
 
+.PHONY: test-ory
+test-ory: export ORY_RECONCILER_INTEGRATION_TESTS=1
+test-ory:
+		go test -v -timeout 5m ./pkg/reconciler/instances/ory/test
```

Comment:
This formatting is strange - looks different when comparing to the rest of the file.